darwin/exporter/formats/mask.py

```diff
@@ -469,3 +486,69 @@ def export(annotation_files: Iterable[dt.AnnotationFile], output_dir: Path, mode
                 writer.writerow([c, f"{palette_rgb[c][0]} {palette_rgb[c][1]} {palette_rgb[c][2]}"])
             else:
                 writer.writerow([c, f"{palette[c]}"])
+
+
+def annotations_exceed_window(annotations: List[dt.Annotation], height: int, width: int) -> bool:
+    """ Check if any annotations exceed the image window
+
+    Args:
+        annotations (List[dt.Annotation]): List of annotations
+        height (int): height of image
+        width (int): width of image
+
+    Returns:
+        bool: True if any annotation exceeds window, false otherwise
+    """
+    for item in annotations:
+        bbox = item.data["bounding_box"]
```

Comment:
Technically `bounding_box` isn't required on polygons. Our backend will always return it, but client-written files may not always include it and this code would crash for it. I don't think we need to complicate this by doing a fallback that computes this bounding box, but maybe skipping annotations that don't have it for this check?

Reply:
I general, I would like files with no bbox to not crash because of this fix. I don't mind not fixing this for them "today" (we can ticket it with lower priority), but I don't want to break anything existing.

Reply:
Just added checks for the field, in both functions that use it, it should default to not extending the window